api-report/runtime-definitions.api.md

```diff
@@ -310,6 +310,16 @@ export interface IInboundSignalMessage extends ISignalMessage {
     type: string;
 }
 
+// @public
+export interface IIncrementalContext {
+    // (undocumented)
+    parentPath: string;
+    // (undocumented)
+    previousSequenceNumber: number;
+    // (undocumented)
+    sequenceNumber: number;
+}
+
```

Comment:
I'd suggest to give more hints (if it's easy to calculate on our side) about "whole situation" :) I.e. maybe add a boolean suggesting if last summary attempt was successful, maybe add boolean to say if last successful attempt was done by this client.
That said, if we provide this data, it has to be filtered to this DDS only. I.e. it's not useful to say that there was a summary attempt (failed or not) if this DDS was not involved.

Which brings us to previousSequenceNumber. I think it should also be filtered through the filter of this DDS. I.e. ideally it reflects sequenceNumber of a summary where this DDS produced anything, and was not just summary reused by upper layer. It might be fine not to do so (if we do not have that info), but it might cause confusion, so we have to document very clearly what this sequence number represents and how DDS authors should think about it.

Reply:
I didn't want the name to get too long, but it might be technically apt to call it `previousSuccessfulSummarySequenceNumber` - the reason being, that number that's being passed down from the `summarizerNode` only gets updated after the summary is successfully acked.